components/list/list-item-generic-layout.js

```diff
@@ -49,12 +60,12 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 				grid-row: 1 / 2;
 			}
 			::slotted([slot="outside-control"]) {
-				width: 40px;
+				width: 2.1rem;
```

Comment:
I recently added style linting to core but I configured it to ignore list components for the moment, as to not create conflicts for you and Maya.  That being said, this will fail linting once enabled - the properties must be order alpha.  Feel free to correct if you like, or I will fix once enables for the list components.

Reply:
I am really happy this is a linting rule. Team Media is really big on it and I try to be as well. I even have an extension that allows me to do it easily. 

I don't mind going through everything in lists after and making sure it is all good.

Reply:
Fixed